README.md

```diff
@@ -12,7 +12,7 @@ Clone the repository
 
 ## Prerequisite
 
-- Flutter 2.10.3
+- Flutter 3.7.9
```

Comment:
Please create a chore task to update this README, it's out of the scope of this PR 🙇

Reply:
@markgravity What did you mean by this change unrelated to this PR? We need to update the Flutter version [here](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/54/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R23) to make this PR is runnable regarding the libraries we used for the UI layout. So it makes sense to update it altogether. 🙏🏻